### PR TITLE
#156882145: Deploy base application to heroku

### PR DESCRIPTION
**Pull request by Spectrumsun**
#### What does this PR do?
Move the code hosted app to Heroku

#### Description of Task to be completed?
A user can access the app on the provided URL in the browser
When a developer raises a Pull Request a review app is automatically created

#### How should this be manually tested?
Paste the URL https://thorin-cfh.herokuapp.com to access the production version
Paste the URL https://thorin-cfh-staging.herokuapp.com to access the development version

#### Any background context you want to provide?
NaN

#### What are the relevant pivotal tracker stories?
[Finishes: #156882145]

#### Screenshots (if appropriate)
NaN

#### Questions:
NaN

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,18 +1,13 @@
 {
-  "name": "thorin-cfh",
+  "name": "cfh",
   "scripts": {
   },
-  "env": {
-    "MONGOHQ_URL": "mongodb://thorin:thorin@ds147659.mlab.com:47659/thorin-cfh",
```
Comment (andela-moe) [inline app.json:6]: Is this the url to the live database? Do you consider it safe to have this here?

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,4 @@ public/lib
 .DS_store
 c4h_keys.txt
 /data
+/.env
```
Comment (andela-moe) [inline .gitignore:7]: Please add a new line to the end of this file.

**Review by andela-moe — COMMENTED** [review]
Please review comments and resolve

File: .gitignore (hunk 2 above)
Comment (Spectrumsun) [reply .gitignore:7]: It's the URL to the staging and testing database. It's stored in the .env file. Heroku auth generated that app.json file and commit it to the staging branch. Will remove before I raise any PR.

**Pull request closed without merging**